src/Blankslate/Blankslate.features.stories.tsx

```diff
@@ -3,7 +3,7 @@ import React from 'react'
 import {Blankslate} from '../Blankslate'
 
 export default {
-  title: 'Drafts/Components/Blankslate/Features',
+  title: 'Private/Components/Blankslate/Features',
```

Comment:
We discussed this in the planning session and seems like it is useful for folks out there (https://github.com/primer/react/pull/3449) do you have any concerns to keep it as draft/experimental?

Reply:
I think we should only export this as experimental if we (primer) want to work on standardising or work on a11y for this component, otherwise it should be created in dotcom/ui/packages?

Reply:
>I think we should only export this as experimental if we (primer) want to work on standardising or work on a11y for this component, otherwise it should be created in dotcom/ui/packages?

For this particular request came from a non-Hubber and I love the idea of exporting components as experimental because it helps/saves time some folks out there. 

>I want to avoid the scenario where folks create a feature request issue or a pull request implementing a feature, and then we don't have the design or a11y context to approve/reject it 😅

Fair enough! I think if we see this is becoming a pattern then we can set some boundaries around it?

Reply:
Sounds like a plan :shipit:

Reply:
@broccolinisoup I'll make the update here to move back to `Drafts`! Will re-request once that's done 👀